server/response/apiresponse.go

```diff
@@ -0,0 +1,44 @@
+package response
+
+type ApiResponse struct {
+	Code int
+	Body ApiResponseBody
+}
+
+type ApiResponseBody struct {
+	Data    interface{} `json:"data,omitempty"`
+	Error   interface{} `json:"error,omitempty"`
+}
+
+type BaseMessage struct {
```

Comment:
this probably isnt necessary

Reply:
on second thought, i definitely would remove this. it just adds extra stuff with no real benefit. it also clutters your Response func calls with `NewBaseMessage()` calls